app/models/office.rb

```diff
@@ -15,4 +15,10 @@ class Office < ActiveRecord::Base
   def managers
     users.where(office_id: id, role: 'manager')
   end
+
+  def business_entities
```

Comment:
Good stuff, we should probably just later extend our tests on the financial report side to include such a case and we're good!